packages/dcos-launch/build

```diff
@@ -1,15 +1,11 @@
 #!/bin/bash
 source /opt/mesosphere/environment.export
 
-# Copy from the ro /pkg/extra to a rw folder.
 mkdir -p /build
 cp -r /pkg/extra/* /build
-# install the lib for export
+cp -r /pkg/src/cerberus/cerberus /build
 cd /build
-pip3 install --no-deps --install-option="--prefix=$PKG_PATH" /pkg/extra/
-# verify the build
-py.test
-# create the installer
+pytest
```

Comment:
Yes. pytest installs both entrypoints. There should actually be no testing in this file, but running in line with the unit tests is not currently possible given the additional dependencies